<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_apmSelect</class>
 <widget class="QDialog" name="form_apmSelect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор АРМ</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pbBookCatalog">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>160</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Электронный Каталог</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGetListReturnBooks">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>350</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbInstruction">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>911</width>
     <height>771</height>
    </rect>
   </property>
   <property name="title">
    <string>Инструкция по использованию АРМ</string>
   </property>
   <widget class="QTextBrowser" name="tbHelpOffline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>891</width>
      <height>721</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Модуль «Электронного каталога»&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/1.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Для поиска интересующего вас издания, необходимо указать в строке ввода, обозначенного как «Поиск», часть или полностью заглавие книги, либо автора, либо серию, в которую входит данная книга. Поисковый запрос начинает обрабатываться, при наличии не менее трех символов из запрашиваемого фильтра:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/2.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Для уточнения местонахождения искомого издания, можно воспользоваться выпадающим списком, расположенного справа от строки поиска. Для сброса уточнения местоположения, нужно выбрать пустую строку.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/3.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Для того, чтобы узнать дополнительные сведения о найденной книге, необходимо выделить книгу щелчком мыши по ней и перейти на вкладку «Описание книги», расположенной слева от списка книг.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/4.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Модуль «Личного кабинета читателя»&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/5.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Для того, чтобы попасть в личный кабинет, вам необходимо воспользоваться читательским билетом. Поднесите ваш ЧБ к, стоящему рядом, ручному сканеру, стороной, на которой нарисован полосковый код. Убедитесь что он появился в поле ввода и нажмите кнопку «Войти». &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;При неверном вводе данных, появится визуальное уведомление и потребуется повторный ввод кода ЧБ.В личном кабинете вам доступны следующие функции:просмотр списка книг и журналов на руках, и количество дней до возврата;просмотр списка взятых ранее книг и журналов;продление срока, на который выдана книга (доступно только 2 продления);&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Список книг выданных изданий и список находящихся на руках;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;С помощью ползунка слева, можно корректировать вид отображения списка (делать больше первый или второй список).&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/6.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;При нажатии кнопки «Продлить», появится окошко со списком изданий, находящихся на руках. Галочками необходимо отметить те книги или журналы, которые вы хотите продлить и нажать кнопку «Продлить».&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/6.png&quot; width=&quot;1024&quot; height=&quot;800&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Библиотекарь получит уведомление о запросе на продление книг и отреагирует соответствующим образом.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>20</width>
     <height>811</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
